[2022-03-18 04:45:57,200] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-10-15T04:00:00+00:00 [queued]>
[2022-03-18 04:45:57,234] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-10-15T04:00:00+00:00 [queued]>
[2022-03-18 04:45:57,236] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:45:57,239] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-03-18 04:45:57,242] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:45:57,296] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_pbp_data> on 2014-10-15 04:00:00+00:00
[2022-03-18 04:45:57,314] {standard_task_runner.py:52} INFO - Started process 429 to run task
[2022-03-18 04:45:57,333] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pbp_bucket_ingestion_dag', 'get_pbp_data', 'scheduled__2014-10-15T04:00:00+00:00', '--job-id', '6246', '--raw', '--subdir', 'DAGS_FOLDER/pbp_ingestion_dag.py', '--cfg-path', '/tmp/tmpgmz119ph', '--error-file', '/tmp/tmp8apl4rdi']
[2022-03-18 04:45:57,336] {standard_task_runner.py:77} INFO - Job 6246: Subtask get_pbp_data
[2022-03-18 04:45:57,562] {logging_mixin.py:109} INFO - Running <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-10-15T04:00:00+00:00 [running]> on host a2ba0547c319
[2022-03-18 04:45:57,814] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pbp_bucket_ingestion_dag
AIRFLOW_CTX_TASK_ID=get_pbp_data
AIRFLOW_CTX_EXECUTION_DATE=2014-10-15T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2014-10-15T04:00:00+00:00
[2022-03-18 04:46:02,829] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 04:46:02,844] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=pbp_bucket_ingestion_dag, task_id=get_pbp_data, execution_date=20141015T040000, start_date=20220318T044557, end_date=20220318T044602
[2022-03-18 04:46:02,918] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-18 04:46:02,965] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
